apps/gallery/index.html

```diff
@@ -98,6 +98,7 @@ <h1 data-l10n-id="gallery" id="fullscreen-title">Gallery</h1>
           <span data-icon="delete"></span>
         </a>
       </gaia-header>
+      <button id="fullscreen-back-button-tiny" type="reset" class="tiny-only" data-icon="back"></button>
```

Comment:
Tested and moving back button on top of frames div hides it behind the image or video poster image in full screen view.